examples/stream-analytics/main.tf

```diff
@@ -0,0 +1,61 @@
+provider "azurerm" {}
```

Comment:
This is a working example. Although, I do have to clarify that I tested creating this entire plan, and also destroying it. I verified that the input for the job that is created is in fact of type "reference".